eng/Versions.props

```diff
@@ -30,7 +30,7 @@
   <PropertyGroup>
     <!-- dotnet/roslyn-analyzers dependencies -->
     <MicrosoftCodeAnalysisAnalyzersVersion>3.3.4-beta1.22559.1</MicrosoftCodeAnalysisAnalyzersVersion>
-    <MicrosoftCodeAnalysisNetAnalyzersVersion>7.0.0-preview1.22559.1</MicrosoftCodeAnalysisNetAnalyzersVersion>
+    <MicrosoftCodeAnalysisNetAnalyzersVersion>8.0.0-preview1.23053.1</MicrosoftCodeAnalysisNetAnalyzersVersion>
```

Comment:
@ViktorHofer, is there something we need to update get this to switch over to using 8.0.* versions?

Reply:
The metadata is already in Version.Details.xml: https://github.com/dotnet/runtime/blob/a6040aed0764e925b1b15449a1028bc52495c9a4/eng/Version.Details.xml#L321-L328

The subscription is stale:
```cmd
C:\git\runtime>darc get-subscriptions --target-repo runtime --source-repo roslyn
https://github.com/dotnet/roslyn-analyzers (.NET 7.0.1xx SDK) ==> 'https://github.com/dotnet/runtime' ('main')
  - Id: 5465c78f-1281-49a8-f9b0-08d9301a7704
  - Update Frequency: EveryDay
  - Enabled: True
  - Batchable: True
  - PR Failure Notification tags:
  - Merge Policies:
    AllChecksSuccessful
      ignoreChecks =
                       [
                         "WIP",
                         "license/cla",
                         "Build Analysis",
                         "runtime-coreclr superpmi-diffs",
                         "runtime-coreclr superpmi-replay",
                         "runtime-libraries enterprise-linux"
                       ]
    NoRequestedChanges
  - Last Build: 20221109.1 (31373ce8529c3d2f6b91e61585872160b0d7d7cd)
```

Reply:
> The subscription is stale

How do we fix that?

Reply:
And it worked. The dependency update just got added to https://github.com/dotnet/runtime/pull/79685

Reply:
Great. Thanks.